codec/encode_test.go

```diff
@@ -74,16 +75,52 @@ var encodeTests = []encodeTest{
 	{val: []*big.Int{big.NewInt(0), big.NewInt(1)}, output: []byte{0x08, 0x00, 0x04}, bytesEncoded: 3},
 }
 
+func setUpStringTests() {
+	// Test strings for various values of n & mode. Also test strings with special characters
+	testString1 := "We love you! We believe in open source as wonderful form of giving." 	// n = 67
+	testString2 := strings.Repeat("We need a longer string to test with. Let's multiple this several times.", 230) 		// n = 72 * 230 = 16560
+	//testString3 := strings.Repeat("We need a longer string to test with. Let's multiple this several times.", 14913081) 	// n = 72 * 14913081 = 1073741832 (> 2^30 = 1073741824)
```

Comment:
Please add comment noting why this commented out (see https://github.com/ChainSafeSystems/go-pre/blob/a89d21793d786cbafec21dd3193ef0870df30945/codec/decode_test.go#L85)